src/App.vue

```diff
@@ -33,13 +33,16 @@
               <option value="strong">Strong (7.5% +)</option>
             </select>
           </BeerFilter>
+
+          <button v-show="!showFavourites" class="btn" @click="toggleFavourites">Show favourites</button>
```

Comment:
This is repetitive. If the only difference is the text, you can use a computed property to display the text you want depending on `showFavourites` value

Reply:
Done - moved to computed property 👍